apps/aevm/src/aevm_eeevm_state.erl

```diff
@@ -239,9 +235,8 @@ do_revert(Us0, Us1, State0) ->
             try
                 %% In Sophia Us1 is a pointer to the actual value.
                 %% The type of the value is always string.
-                HeapValue = aeso_data:heap_value(maps(State0), Us1, mem(State0)),
-                {ok, Out} = aeso_data:heap_to_binary(string, get_store(State0), HeapValue),
-                set_out(Out, State0)
+                {ok, Out, GasUsed} = heap_to_binary(string, Us1, State0),
```

Comment:
Why not get rid of the `try...catch` here as well, and simply use a `case` switch?

Reply:
I wanted to leave something for the "review use of `try...catch`"-ticket..? We should though.